WORKSPACE

```diff
@@ -25,6 +25,13 @@ git_repository(
     tag = "v0.9.0",
 )
 
+new_git_repository(
```

Comment:
`new_git_repository` is misleading. Rename it to `third_party_repo` or something alike.

Reply:
`new_git_repository` is bazel command.

Reply:
Should we use `new_git_repository` or `git_repository`?

Reply:
The `git_repository` should have `WORKSPACE` inside repository.